src/containers/button/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement } from 'react';
+import { GestureResponderEvent, TouchableOpacity, Text } from 'react-native';
+import styled from 'styled-components';
+import colors from '../../theme/colors';
+
+interface IButton {
```

Comment:
Naming interfaces with ```I``` it's bad practice in Typescript. You can name this like a ButtonProps